package-lock.json

```diff
@@ -18973,6 +18973,18 @@
 						"hoek": "2.x.x",
 						"joi": "6.x.x",
 						"wreck": "5.x.x"
+					},
```

Comment:
Please don't commit your `package-lock` changes

Reply:
Sorry about that. I didn't knew it back then. 
Actually I have deleted the previous fork so I can't make changes to it. Can I open a new Pull request for this?